sample_apps/rust/examples/query_sample.rs

```
@@ -0,0 +1,220 @@
+        SELECT INTERPOLATE_LINEAR(\
+        CREATE_TIME_SERIES(time, ROUND(cpu_utilization, 2)), \
+        SEQUENCE(min(time), max(time), 10s)) AS interpolated_cpu_utilization \
+        FROM {database_name}.{table_name} \
+        WHERE hostname = '{hostname}' AND measure_name = 'metrics' \
+        GROUP BY hostname \
+        )\
+        SELECT REDUCE(interpolated_cpu_utilization, \
+        DOUBLE '0.0', \
+        (s, x) -> s + 1, \
+        s -> s) AS count_cpu \
+        FROM time_series_view"
+    );
+
+    // 12. Find the average CPU utilization for a specific EC2 host, filling in the missing values using linear interpolation.
+    let query_12 = format!(
+        "WITH time_series_view AS (\
+        SELECT INTERPOLATE_LINEAR(CREATE_TIME_SERIES(time, ROUND(cpu_utilization, 2)), \
+        SEQUENCE(min(time), max(time), 10s)) AS interpolated_cpu_utilization \
+        FROM {database_name}.{table_name} \
+        WHERE hostname = '{hostname}' AND measure_name = 'metrics' \
+        GROUP BY hostname) \
+        SELECT REDUCE(interpolated_cpu_utilization, \
+        CAST(ROW(0.0, 0) AS ROW(sum DOUBLE, count INTEGER)), \
+        (s, x) -> CAST(ROW(x.value + s.sum, s.count + 1) AS ROW(sum DOUBLE, count INTEGER)), \
+        s -> IF(s.count = 0, NULL, s.sum / s.count)) AS avg_cpu \
+        FROM time_series_view"
+    );
+
+    // 13. Running a query with multiple pages
+    let query_13 = format!("SELECT * FROM {database_name}.{table_name} LIMIT {query_limit}");
+
+    let queries = [
+        query_1, query_2, query_3, query_4, query_5, query_6, query_7, query_8, query_9, query_10,
+        query_11, query_12, query_13,
+    ];
+
+    for (i, query) in queries.iter().enumerate() {
+        let msg = format!("Running Query_{} : {}", i + 1, query);
+        println!("{}", msg);
+        let _ = query_common::write(f, msg.to_string());
+        let _ = query_common::run_query(query.to_string(), &client, f, max_rows).await;
```

You explicitly ignore errors? Why?

---

Changed to not ignore errors.